get-started/versioning-availability.md

```diff
@@ -7,208 +7,53 @@ products:
 description: Learn how Elastic handles versioning and feature availability in the docs. Find the product versions that are supported, how to read availability badges, and...
 ---
 
-# Versioning and availability in Elastic Docs
+# Versioning and availability
 
-Learn how Elastic Docs handles versioning, feature availability, and how to find the right documentation for your deployment type and product version. Find answers to common questions about the Elastic Stack versioning and confidently navigate our continuously updated documentation.
+Learn how Elastic products are versioned, the lifecycle of features, and how to find the right documentation for your deployment type and product version. Find answers to common questions about the versioning and confidently navigate our continuously updated documentation.
 
-## Frequently asked questions
```

Comment:
be cautious about removing this stuff. most of this was added to stop bleeding stemming from the migration.